batches/aapb-collaboration-7-leftover.txt

```diff
@@ -1,4 +1,13 @@
-# see https://github.com/clamsproject/aapb-annotations/issues/79 for more info
+# --------------------------------------------------------------------------------
+# Originally, a set of about 4000 videos was selected for slate annotation. This 
+# batch has the GUIDs that were not include in batch aapb-collaboration-7, which
+# contains the GUIDs of assets that were annotated.
+# 
+# See https://github.com/clamsproject/aapb-collaboration/issues/79 for more info.
+#
+# Note that the issue identifier on this batch does not match the related issue,
+# probably due to a human error somewhere in the loop.
```

Comment:
Yeah, definitely was my bad from 16224dbe59e644002c9317a985e8607c8cc5e6fb. Probably a typo...

Reply:
Hmmm, I am mixing up my comments between leftover and non-leftover, but in general I think all that was needed is to turn 79 into 7 and remove the note that something was awry. Which I did